CLIENT-SPECIFICATION.md

```diff
@@ -1,7 +1,7 @@
 <!-- markdownlint-disable MD007 MD013 MD024-->
 # tldr-pages client specification
 
-**Current Specification Version:** 2.3
+**Current Specification Version:** Unreleased
```

Comment:
Please leave it at 2.3 for the moment. We will bump it once we release a new version.

Reply:
We did the same during the last release cycle. I thought to do the same this time.